source/frontend/src/assets/scss/App.scss

```diff
@@ -95,6 +95,16 @@ a {
     font-size: 1.4rem;
   }
 
+  .searchWrapper {
```

Comment:
This presents a more consistent view for the multi-selects in the application, which were previously slightly larger than or normal inputs.